libs/providers/ofrep-web/project.json

```diff
@@ -68,7 +68,8 @@
             "input": "./libs/providers/ofrep-web",
             "output": "./"
           }
-        ]
+        ],
+        "updateBuildableProjectDepsInPackageJson": true
```

Comment:
This now automatically generates a dependency entry for us for core (so I've removed the corresponding peer).

Reply:
I have never used this option, but the doc reads like this making sense!
Have you tried it?

Reply:
We actually currently use for for flagd/flagd-web to automatically pull in flagd-core, so I know it works!

Reply:
Sounds great!